hi-ml-multimodal/README.md

```diff
@@ -17,7 +17,7 @@ The notebook can also be run on [Binder][4], without the need to download any co
 The latest version can be installed using `pip`:
 
 ```console
-pip install "git+https://github.com/microsoft/hi-ml.git#subdirectory=hi-ml-multimodal"
+pip install --upgrade hi-ml-multimodal
```

Comment:
@fepegar This assumes that a new package is going to be released at the same time as we merge the PR in. I guess you are on it?

Reply:
Sounds good. As soon as we merge, I'll publish. It'll take me a couple of minutes.